src/main/webapp/resources/js/components/ant.design/PagedTable/PagedTableContext.js

```diff
@@ -96,14 +96,15 @@ function PagedTableProvider({
       sortColumn: state.column || "createdDate",
       sortDirection: state.order || "descend",
       search: state.search,
-      filters: state.filters
+      filters: state.filters,
     }).then(({ dataSource, total }) => {
+      console.log(dataSource);
```

Comment:
Remove logging

Reply:
Removed in 8c8ab3bf8a0eda29a11344aa5c32ffe59c857335